laalaa/apps/advisers/tasks.py

```diff
@@ -114,11 +114,7 @@ def __init__(self):
         self.temp_dir = tempfile.mkdtemp()
         self.progress = {"task": "initialising"}
 
-    def run(self, xlsx_file, record=None, *args, **kwargs):
-        self.record = record
-
-        self.update_state(state="INITIALIZING", meta=self.progress)
-
+    def prepare_tables(self, xlsx_file, *args, **kwargs):
```

Comment:
What do you think about `truncate_and_upload_data_tables_from_xlsx`? Deliberately long named to highlight the dangers :)